app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
+  validates :name, presence: { message: 'Name field can not left blank' }
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Kindly write _**validation**_ tests for these attributes. Remember to include test examples for the model being `not_valid` and test examples for the presence of attributes. 

### `not_valid` case 
```ruby on rails
 it 'should be invalid if name column is blank' do
      subject.name = nil
      expect(subject).to_not be_valid
    end
```

### `validate_presence` of attribute

```ruby on rails
describe 'validations' do
  it { should validate_presence_of(:name) }
end

```
```ruby on rails
 it 'should be a valid user' do
      expect(subject).to be_valid
    end
```